duckduckgo/iOS
# automate screenshots

Reviewer: Mia, Jeffrey
Asana: https://app.asana.com/0/414235014887631/390649588301838/f
and fixes a bug for: https://app.asana.com/0/414235014887631/413609835171504/f
CC:

**Description**:

Automates screenshot creation for a range of iPhones and iPads

**Steps to test this PR**:

1. With fastlane installed run `fastlane screenshots`
2. Once complete a browser will open with a html file showing each of the screenshots

3. Launch the app normally and explicitly enter a search url which overrides the location parameter, e.g. https://duckduckgo.com/q=privacy&kl=ie-en should show results for Ireland 
4. After the page loads the URL should contain the mobile stats params


###### Reviewer Checklist:


###### PR DRI Checklist:
